src/main/java/org/openpnp/machine/reference/ReferenceNozzle.java

```diff
@@ -171,7 +161,7 @@ public void place() throws Exception {
     }
     
     private ReferenceNozzleTip getUnloadedNozzleTipStandin() {
-        for (NozzleTip nozzleTip : this.getNozzleTips()) {
+        for (NozzleTip nozzleTip : this.getCompatibleNozzleTips()) {
```

Comment:
Since Nozzle no longer owns NozzleTips, I changed this to getCompatibleNozzleTips. It's not an exact analog of what you were doing before, since now the standin might be shared.

Reply:
This is correct, with the additions I made.